packages/deploy-helpers/deploy-logger.js

```diff
@@ -2,28 +2,57 @@ const flatten = require('truffle-flattener')
 const mkdirp = require('mkdirp')
 const fs = require('fs')
 const path = require('path')
-
-const truffleConfig = require('@aragon/truffle-config-v4/truffle-config')
+const findUp = require('find-up')
 
 const FLATTEN_DIR = './flattened_contracts'
 
-module.exports = async (instance, { verbose = true, flattenContracts = true } = {}) => {
+const getTruffleConfig = () => {
+  try {
+    let truffleConfigPath = findUp.sync('truffle.js')
+    if (fs.existsSync(truffleConfigPath)) {
+      const truffleConfig = require(truffleConfigPath)
+      return truffleConfig
+    }
+    truffleConfigPath = findUp.sync('truffle-config.js')
+    if (fs.existsSync(truffleConfigPath)) {
+      const truffleConfig = require(truffleConfigPath)
+      return truffleConfig
+    }
+    const truffleConfig = require('@aragon/truffle-config-v4')
```

Comment:
Not sure about defaulting to this, people will still need to install the package.